src/pymortests/demos.py

```diff
@@ -115,6 +115,11 @@
     ('unstable_heat', [50, 10]),
 )
 
+DD_MOR_ARGS = (
+    ('dd_parabolic_heat', [0.01, 50, 10]),
```

Comment:
You can also change this in the PR title.

```suggestion
    ('dd_parametric_heat', [0.01, 50, 10]),
```

Reply:
whoops